doc/fern.txt

```diff
@@ -421,6 +421,15 @@ VARIABLE						*fern-variable*
 
 	Default: 0
 
+*g:fern#disable_drawer_universal*
+	Set 1 to disable the universal fern trees for left and right position
+        respectively.
+
+        If enabled, there is only one left or right drawer across the all the windows
+        for a given VIM instance.
+
+        Default: 1
```

Comment:
This new behavior should be "opt-in" but the default value of variable should be 0.

So maybe `g:fern#disable_drawer_tabpage_isolation` or whatever is preferable.

Reply:
Done